Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN chmod +x robo.phar && mv robo.phar /usr/local/bin/robo
 RUN apt-get install -y libxslt-dev
 RUN docker-php-ext-install bcmath xsl
 
+RUN apt-get install -y mariadb-client
```

Comment:
Shall we remove `RUN apt-get install -y sqlite3` ?

Reply:
Right now, it's still required for the kernel test database. At some point, I may change it so that we _only_ do a site install for behat, but for now I'd like to focus on "working" vs "fast".